### ISSUE TITLE
I cannot view the video generated from the test on dashboard cypress

### ISSUE THREAD

**Issue by lucasmp2001** [cypress-io/cypress#17746]
### Current behavior

i'm trying to get the videos , and I'm want to see the videos that generate on the test dashboard on cypress, but it gives an error, or it only takes 1 second of video and I can't open it...

the error that appears is the following:

`The media could not be loaded, either because the server or network failed or because the format is not supported`

does anyone have any idea what it can be?

(maybe i'm using a old version, and this is already fixed?)

### Test code to reproduce

![image](https://user-images.githubusercontent.com/74330563/129569620-61dc9282-f1e3-415b-8ef8-f40b52c8f9c6.png)

![image](https://user-images.githubusercontent.com/74330563/129569781-cb7a6d00-9787-4909-be19-8a7598db05a6.png)

### Cypress Version

5.6.0

### Other

`cypress.json`

{
  "projectId": "******",
  "integrationFolder": "cypress/integration/axulus_ui_tests",
  "reporter": "junit",
  "reporterOptions": {
    "mochaFile": "tests/test-output-[hash].xml",
    "toConsole": true,
    "attachments": true
  },
  "video": true,
  "videoUploadOnPasses": false,
  "pluginsFile": "cypress/plugins/index.js",
  "supportFile": "cypress/support/index.js",
  "viewportWidth": 1920,
  "viewportHeight": 1500,
  "pageLoadTimeout": 20000
}

**Comment by jennifer-shehane** [comment]
Since you are using an older version of Cypress, I'd suggest updating to the [current version](https://on.cypress.io/changelog) and let us know if this is still happening for you. We've made a lot of improvements since 5.6.

Also, can you print the stdout log of the run? Usually we see this when the video processing failed for some reason and this error is printed in the terminal during `cypress run`.

**Comment by lucasmp2001** [comment]
right, i will do this update on cypress, maybe this error can be fixed.

and, on the log, is showing this:

```
────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                    
  Running:  admin/users-management/user-roles.spec.js                                       (5 of 5)
  Estimated: 4 minutes, 28 seconds
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="5" failures="5">
  <testsuite name="Root Suite" timestamp="2021-08-16T12:45:02" tests="0" file="cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js" failures="0" time="0">
  </testsuite>
  <testsuite name="User Roles (@admin)" timestamp="2021-08-16T12:45:02" tests="5" failures="5" time="0">
    <testcase name="User Roles (@admin) As a Portfolio Manager, should invite a new user" time="0" classname="As a Portfolio Manager, should invite a new user">
      <failure message="Timed out retrying: `cy.click()` failed because this element is detached from the DOM.

`&lt;button _ngcontent-aot-c332=&quot;&quot; pbutton=&quot;&quot; pripple=&quot;&quot; type=&quot;button&quot; class=&quot;p-button-sm p-button p-component&quot; disabled=&quot;&quot; data-cy=&quot;invitation-modal__send&quot;&gt;...&lt;/button&gt;`

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  &gt; `cy.get()`

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add &apos;guards&apos; which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom" type="CypressError"><![CDATA[CypressError: Timed out retrying: `cy.click()` failed because this element is detached from the DOM.

`<button _ngcontent-aot-c332="" pbutton="" pripple="" type="button" class="p-button-sm p-button p-component" disabled="" data-cy="invitation-modal__send">...</button>`

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  > `cy.get()`

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add 'guards' which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom
    at $Cy.ensureAttached (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:161173:24)
    at runAllChecks (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:150104:12)
    at retryActionability (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:150163:16)
    at tryCatcher (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:10325:23)
    at Function.Promise.attempt.Promise.try (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:7599:29)
    at tryFn (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:165556:21)
    at whenStable (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:165594:12)
    at https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:165089:16
    at tryCatcher (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:10325:23)
    at Promise._settlePromiseFromHandler (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:8260:31)
    at Promise._settlePromise (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:8317:18)
    at Promise._settlePromise0 (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:8362:10)
    at Promise._settlePromises (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:8442:18)
    at Promise._fulfill (https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:8386:18)
    at https://staxwebstag.z1.web.core.windows.net/__cypress/runner/cypress_runner.js:10000:46
From Your Spec Code:
    at Context.eval (https://staxwebstag.z1.web.core.windows.net/__cypress/tests?p=cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js:40:50)]]></failure>
    </testcase>
    <testcase name="User Roles (@admin) First Login of user invited, changing the password and login with new password created" time="0" classname="First Login of user invited, changing the password and login with new password created">
      <failure message="expected 1 to not equal 1" type="AssertionError"><![CDATA[AssertionError: expected 1 to not equal 1
    at Context.eval (https://staxwebstag.z1.web.core.windows.net/__cypress/tests?p=cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js:50:31)]]></failure>
    </testcase>
    <testcase name="User Roles (@admin) As a Portfolio Manager, should be able to reset a user password" time="0" classname="As a Portfolio Manager, should be able to reset a user password">
      <failure message="Timed out retrying: Expected to find element: `.pi-pencil`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying: Expected to find element: `.pi-pencil`, but never found it.
    at Context.eval (https://staxwebstag.z1.web.core.windows.net/__cypress/tests?p=cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js:110:8)]]></failure>
    </testcase>
    <testcase name="User Roles (@admin) Doing Login after password being reseted" time="0" classname="Doing Login after password being reseted">
      <failure message="expected 1 to not equal 1" type="AssertionError"><![CDATA[AssertionError: expected 1 to not equal 1
    at Context.eval (https://staxwebstag.z1.web.core.windows.net/__cypress/tests?p=cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js:122:31)]]></failure>
    </testcase>
    <testcase name="User Roles (@admin) As a Portfolio Manager, should be able to delete a user" time="0" classname="As a Portfolio Manager, should be able to delete a user">
      <failure message="Timed out retrying: Expected to find element: `.pi-pencil`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying: Expected to find element: `.pi-pencil`, but never found it.
    at Context.eval (https://staxwebstag.z1.web.core.windows.net/__cypress/tests?p=cypress/integration/axulus_ui_tests/admin/users-management/user-roles.spec.js:153:8)]]></failure>
    </testcase>
  </testsuite>
</testsuites>

  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        5                                                                                │
  │ Passing:      0                                                                                │
  │ Failing:      5                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  5                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     4 minutes, 28 seconds                                                            │
  │ Estimated:    4 minutes, 28 seconds                                                            │
  │ Spec Ran:     admin/users-management/user-roles.spec.js                                        │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


  (Screenshots)

  -  /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.j     (1050x888)
     s/User Roles (@admin) -- As a Portfolio Manager, should invite a new user (faile               
     d).png                                                                                         
  -  /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.j     (1050x888)
     s/User Roles (@admin) -- First Login of user invited, changing the password and                
     login with new password created (failed).png                                                   
  -  /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.j     (1050x888)
     s/User Roles (@admin) -- As a Portfolio Manager, should be able to reset a user                
     password (failed).png                                                                          
  -  /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.j     (1050x888)
     s/User Roles (@admin) -- Doing Login after password being reseted (failed).png                 
  -  /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.j     (1050x888)
     s/User Roles (@admin) -- As a Portfolio Manager, should be able to delete a user               
      (failed).png                                                                                  


  (Video)

  -  Started processing:  Compressing to 32 CRF                                                     
  -  Finished processing: /home/vsts/work/1/s/cypress/videos/admin/users-management/u    (0 seconds)
                          ser-roles.spec.js.mp4                                                     


  (Uploading Results)

  - Done Uploading (1/6) /home/vsts/work/1/s/cypress/videos/admin/users-management/user-roles.spec.js.mp4
  - Done Uploading (2/6) /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.js/User Roles (@admin) -- As a Portfolio Manager, should invite a new user (failed).png
  - Done Uploading (3/6) /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.js/User Roles (@admin) -- As a Portfolio Manager, should be able to reset a user password (failed).png
  - Done Uploading (4/6) /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.js/User Roles (@admin) -- As a Portfolio Manager, should be able to delete a user (failed).png
  - Done Uploading (5/6) /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.js/User Roles (@admin) -- Doing Login after password being reseted (failed).png
  - Done Uploading (6/6) /home/vsts/work/1/s/cypress/screenshots/admin/users-management/user-roles.spec.js/User Roles (@admin) -- First Login of user invited, changing the password and login with new password created (failed).png
```

**Comment by lucasmp2001** [comment]
I have a question

In my tests, there is a specific one that I do two process in a single test, and I use the Puppeteer within the cypress test, and in this test, there is a step that runs on this Puppeteer that is failing, but when I run on mine machine does not show any error, but in Pipeline yes, and i want to capture this error on video recording.

Can cypress capture the puppeteer's screen doing the actions, even though it's doing an action over the cypress, and I'm calling it within the test?

@jennifer-shehane

**Comment by emilyrohrbough** [comment]
@lucasmp2001 This issue sounds like https://github.com/cypress-io/cypress/issues/2939, which was fixed and released in Cypress 10.8.0. Closing this a fixed. Please comment or open a new issue if this issue persists.
